src/ontology/omo-edit.owl

```diff
@@ -1398,7 +1398,10 @@ No imports</obo:IAO_0000115>
         <rdf:type rdf:resource="http://purl.obolibrary.org/obo/IAO_0000225"/>
         <obo:IAO_0000111 xml:lang="en">failed exploratory term</obo:IAO_0000111>
         <obo:IAO_0000115 xml:lang="en">The term was used in an attempt to structure part of the ontology but in retrospect failed to do a good job</obo:IAO_0000115>
+        <obo:IAO_0000116 xml:lang="en">Terms created with the expectation of being permanent should not be deprecated with this justification Terms created as placeholders should use the placeholder removed justification.</obo:IAO_0000116>
         <obo:IAO_0000117 xml:lang="en">Person:Alan Ruttenberg</obo:IAO_0000117>
+        <dc:contributor rdf:resource="https://orcid.org/0000-0001-6875-5360"/>
```

Comment:
I suggest using always the same style (nominal vs verbal, with article or without...
Suggestion for Line  426 (I have no writing persmissions):
```
        <obo:IAO_0000115 xml:lang="en">The term was created to ease viewing/sort terms for development purpose, but will not be included in a release</obo:IAO_0000115>

```